app/src/main/java/nextstep/signup/MainActivity.kt

```diff
@@ -3,44 +3,156 @@ package nextstep.signup
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.unit.sp
 import nextstep.signup.ui.theme.SignupTheme
 
 class MainActivity : ComponentActivity() {
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
         setContent {
             SignupTheme {
-                // A surface container using the 'background' color from the theme
-                Surface(
-                    modifier = Modifier.fillMaxSize(),
-                    color = MaterialTheme.colorScheme.background
-                ) {
-                    Greeting("Android")
-                }
+              SignUpScreen()
             }
         }
     }
 }
 
 @Composable
-fun Greeting(name: String, modifier: Modifier = Modifier) {
+fun SignUpScreen() {
+    Surface(
+        modifier = Modifier
+            .fillMaxSize()
+            .padding(
+                start = 32.dp,
+                top = 50.dp,
+                end = 32.dp,
+            ),
+        color = MaterialTheme.colorScheme.background,
+    ) {
+        var username by rememberSaveable { mutableStateOf("") }
+        var email by rememberSaveable { mutableStateOf("") }
+        var password by rememberSaveable { mutableStateOf("") }
+        var passwordConfirm by rememberSaveable { mutableStateOf("") }
+
+        Column(
+            horizontalAlignment = Alignment.CenterHorizontally,
+        ) {
+            HeadLine(stringResource(R.string.sign_up_headline))
+            Spacer(modifier = Modifier.height(36.dp))
+            SignUpTextField(
+                value = username,
+                onValueChange = { username = it },
+                label = stringResource(R.string.sign_up_username_label),
+            )
+            Spacer(modifier = Modifier.height(36.dp))
+            SignUpTextField(
+                value = email,
+                onValueChange = { email = it },
+                label = stringResource(R.string.sign_up_email_label),
+                keyboardOptions = KeyboardOptions(keyboardType = KeyboardType.Email)
+            )
+            Spacer(modifier = Modifier.height(36.dp))
+            SignUpTextField(
+                value = password,
+                onValueChange = { password = it },
+                label = stringResource(R.string.sign_up_password_label),
+                keyboardOptions = KeyboardOptions(keyboardType = KeyboardType.Password),
+                visualTransformation = PasswordVisualTransformation()
+            )
+            Spacer(modifier = Modifier.height(36.dp))
+            SignUpTextField(
+                value = passwordConfirm,
+                onValueChange = { passwordConfirm = it },
+                label = stringResource(R.string.sign_up_password_confirm_label),
+                keyboardOptions = KeyboardOptions(keyboardType = KeyboardType.Password),
+                visualTransformation = PasswordVisualTransformation()
+            )
```

Comment:
같은 컴포넌트 간에 같은 정도의 간격을 주기 위해 Spacer를 반복적으로 호출하는 방법도 있겠지만, 개인적으로는 코드를 읽는데 피로감이 들 것 같습니다.

Column 및 Row의 Arrangement에는 `spacedBy()`라는 메소드가 존재합니다.
TextField들끼리 같은 간격을 가져야 한다면, 저는 TextField끼리 하나의 컴포넌트로 분리하여 아래와 같이 간격을 설정할 것 같습니다.
```suggestion
Column(
    modifier = modifier,
    verticalArrangement = Arrangement.spacedBy(36.dp),
) {
    SignUpTextField(/* */)
    
    SignUpTextField(/* */)    
    
    SignUpTextField(/* */)
        
    SignUpTextField(/* */)
}
```

Reply:
spacedBy() 기억하고 잘 사용하겠습니다. 훨씬 가독성 좋네요! 🫡  [1496edb](https://github.com/woowacourse/android-signup/pull/18/commits/1496edbae41630abe529e0abbb302fabefc97664)